src/app/renderer.js

```diff
@@ -101,6 +108,12 @@ Renderer.prototype.contracts = function (data, source) {
 
     if (contract.bytecode) {
       $contractOutput.append(uiHelper.preRow('Web3 deploy', uiHelper.gethDeploy(contractName.toLowerCase(), contract['interface'], contract.bytecode), 'deploy'))
+
+      // check if there's a metadata hash appended
+      var metadataHash = retrieveMetadataHash(contract.bytecode)
+      if (metadataHash) {
+        $contractOutput.append(uiHelper.tableRow('Metadata location', 'bzzr://' + metadataHash))
```

Comment:
We could use `'http://swarm-gateways.net/bzzr:/' + metadataHash` until we have support for the bzzr protocol